flake.nix

```diff
@@ -82,9 +82,7 @@
         });
 
         configureFlags =
-          [
-            "CXXFLAGS=-I${lib.getDev rapidcheck}/extras/gtest/include"
```

Comment:
There is more `CXXFLAGS` in this file thanks to a backported fix.

Reply:
Done. No more `CXXFLAGS` in any `.nix` now.